### DISCUSSION TITLE
unsupported texture map image format for YCB dataset and more

### DISCUSSION THREAD

**Discussion by shivanimall** [bulletphysics/bullet3#3845] [Q&A] [answered]
I have texture map images from datasets such as YCB dataset, 3Dmodelhaven

However, I get this following error when loading it with pybullet:
`b3Printf: Unsupported texture image format` any resolution for this? using a 8-bit_Diffuse texture map
attached an example image here: 
https://drive.google.com/file/d/1VtCvE5mIGJ9U077ksyPoQc_lgJL4Upfx/view?usp=sharing

Also, noting that using these works fine with blender.

**Comment by shivanimall** [comment, marked as answer]
https://github.com/sea-bass/ycb-tools/issues/3

**Comment by erwincoumans** [comment]
PyBullet uses stb image to load textures, you may need to convert textures. See https://github.com/bulletphysics/bullet3/tree/master/examples/ThirdPartyLibs/stb_image
